examples/async/async.js

```diff
@@ -166,29 +166,17 @@ document.addEventListener('click', (e) => {
   if(e.target.matches('.button-click')) {
 
     aa('click', {
+      eventName: "hit-clicked",
+      indexName: process.env.INDEX_NAME,
       objectID: e.target.getAttribute('data-objectid'),
       position: parseInt(e.target.getAttribute('data-position'))
```

Comment:
Careful inside the example because I don't think the provided position is the correct one. The one we get here is the relative but IIRC the one we have to provide is the absolute, correct?

Reply:
Don't use the built-in widgets but only the connectors.

Reply:
opened a PR on top for that https://github.com/algolia/search-insights.js/pull/31

Reply:
Note that i'm not using connectors but transformData as discussed with Marie